test/model/create.test.js

```diff
@@ -210,7 +210,8 @@ const _         = require('lodash');
                     .src(Support.resolveSupportPath('tmp', 'models'))
                     .pipe(helpers.readFile('user.js'))
                     .pipe(helpers.ensureContent(targetContent))
-                    .pipe(helpers.teardown(done));
+                    .pipe(helpers.teardown(done))
+                    .pipe(helpers.ensureContent('.associate'));
```

Comment:
`.pipe(helpers.ensureContent('.associate'))`, Should come before `teardown`, I think tests will fail